tasks/Moving offices (complex task).md

```diff
@@ -0,0 +1,65 @@
+
+Card with information (8 different cards):
+(1)The company owns a total of 12 desks 
+(2)The company owns a total of 48 chairs
+(3)The company owns a total of 12 typewriters
+(4)The company owns a total of 15 filing-cabinets
+(5)One truckload can take 12 typewriters
+(6)One truckload can take 3 desks 
+(7)One truckload can take 5 filing-cabinets
+(8)One truckload can take 24 chairs
+
+# Procedure
+## Steps
+
+1. The subjects are told that the experiment is to investigate how fast groups can solve certain abstract problems and there are five problems to be solved.
+2. Each subject is given one of the problem cards and some information cards where the information was equally divided over the group members. For completion of the group task each subject has to know the problem solution.
+3. The subjects are seated around a circular table, so that each is separated from the next by a vertical partition from the center to the table's edge.
+4. The subjects are supplied with blank message cards whose colors matched those of their work spaces for communication to the other members of the group.
+5. When the group has solved the problems, each subject is asked to fill out a questionnaire, in which mainly measurements of satisfaction and leadership were made.
+
+## Roles 
+
+Each participant completed the same task following instructions.
+
+## Instructions
+
+Instruction written by CSSLab:
+
+Thank you for participating in our experiment today. We want to investigate how fast groups can solve certain abstract problems. Five problems had to be solved. We would have preferred more problems, to allow for the complete development of the decision structure, but this was the maximum possible in the time that we have available.
+
+Each of you will be given a card with information on it that will help solve the problem and you will need to work together to find a solution. In order to communicate with your fellow team members you can pass notes through slots permitting you to push written message cards to the persons on either side of of you.
+
+To allow for communication to the other members of the group, a four- layered quadrilateral box is at the center of the table. The box was placed so that the partitions just touched each of the four points of the quadrangle. Each of the four resulting work spaces was painted a different color. You have been supplied with blank message cards whose colors matched those of your work spaces. Any message sent from a booth had to be on a card of the booth's color.
+
+Now for the problem:
+
+A small company is moving from one office-building to another. It must move four kinds of equipment: (a) chairs, (b) desks, (c) filing-cabinets and (d) typewriters. How many trucks are needed to make the move in one trip?
+
+When finished solving the problem:
+
+Please fill out this short questionnaire.
```

Comment:
I actually just realized, do we have the questions in the questionnaire? If so, let's add them to the stimuli? 

Missed this the first time around!

Reply:
We don't have the questions in the questionnaire in this paper!